packages/program/src/functions/base-invocation-scope.ts

```diff
@@ -359,7 +363,7 @@ export class BaseInvocationScope<TReturn = any> {
    *
    * @returns The result of the function invocation.
    */
-  async call<T = TReturn>(): Promise<FunctionInvocationResult<T>> {
+  async call<T = TReturn>(): Promise<SubmitResult<T>> {
```

Comment:
Have we gone for `submit` nomenclature over `call` in aid of #1452?

Reply:
@danielbate that was my initial thinking. But I've changed it [here](https://github.com/FuelLabs/fuels-ts/pull/2652/commits/4b73be269c240f4647a3977fe9504a067341341c)